### DISCUSSION TITLE
Connect code option in radial wheel

### DISCUSSION THREAD

**Discussion by Dovakillz** [gameserverapp/Platform#921] [Suggestions]
As im sure we can all agree with the issues consoles have getting the connect code if we could get a button on the radial wheel to do this to get the code would probably be alot better.

**Comment by babsgosgens** [comment]
I like that suggestion, if that is a viable option, that would make it a lot easier. Only issue to consider is console's inexperience with the wheel, they would still struggle. Maybe it can be an extra option, instead of a full replacement?

**Reply by Dovakillz** [reply]
Definitely dont want to replace the original method.

**Reply by TwistedBobRoss** [reply]
I think this is a fantastic idea.  I would leave the text command in place as usual.  But what about modified pause menu UI? Other mods do it like the backup inventory saver.  Just simply add a button to the pause menu that generates the code on command.  I do not know what is involved with achieving that or what else it may impact but as far as ease of use for a user, seems to be a viable option.  I also like the radial menu idea.  I dont know how difficult it is for Xbox users, but on PSN radials seem easy enough.

Most of my connect code player issues is from Xbox.  Not nearly as frequent on the PSN players and never on PC players it seems lol
